go/host/p2p/metrics.go

```diff
@@ -0,0 +1,60 @@
+package p2p
```

Comment:
Basically this the convention we want to have?
A "metrics.go" file in the packages( functionalities) which we want to monitor?

Reply:
Yeap! Each package is now responsible for registering it's own metrics. 
The alternative was to have a central metrics package where metrics were defined but that felt like it was going to become bloated very quickly.